File: back-end/src/main/java/com/example/edicionlimitada/demo/model/Cliente.java
package com.example.edicionlimitada.demo.model;
import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

import com.fasterxml.jackson.annotation.JsonProperty;

import lombok.Getter;
import lombok.Setter;




@Document(collection = "cliente")
@Getter
@Setter
public class Cliente {
    
    @Id
    @JsonProperty("_id")
    private String id;
    private String nombre;
    private String email;
    private String contrasena;
    private String comuna;
    private String region;

}

File: android-app/app/src/main/java/com/vivitasol/carcasamvvm/model/Cliente.kt
package com.vivitasol.carcasamvvm.model

import com.google.gson.annotations.SerializedName

data class Cliente(
    @SerializedName("_id")
    val id: String,
    val nombre: String,
    val email: String,
    val contrasena: String?, // ¡CORREGIDO! El campo ahora es opcional
    val comuna: String,
    val region: String
)
